colossalai/inference/inferenceData.py

```diff
@@ -2,6 +2,10 @@
 from dataclasses import dataclass
 from typing import Dict, List, Set
 
+"""
+The abstraction of request and sequence are defined here.
+"""
```

Comment:
This file's name is not consistent with Python convention, `data.py` will be enough.

Reply:
fixed